extensions/amp-iframe/0.1/validator-amp-iframe.protoascii

```diff
@@ -72,6 +72,8 @@ tags: {  # <amp-iframe>
     name: "srcdoc"
     mandatory_oneof: "['src', 'srcdoc']"
   }
+  # <amp-bind>
+  attrs: { name: "[src]" }
```

Comment:
similar problem here, one of 'src' or 'srcdoc' is required here. Same sol'n recommended.